docs/_includes/extra/script.js

```diff
@@ -2,6 +2,18 @@
 *  doesn't work due to fixed body height when menu is toggled.
 *  This script intercepts clicks on links, toggles the menu off
 *  and performs the anchor navigation. */
+
+window.addEventListener("load", function () {
+    let href = window.location.pathname;
+    const hash = window.location.hash;
+    if (hash!=="") {
+        href=hash
+    }
+    const element = document.querySelector(`a[href="${href}"]`);
+    element.scrollIntoView({ behavior: "smooth" });
```

Comment:
I think it's worth adding a check that the element will be found (not undefined), otherwise in some cases, it may lead to an error.

Reply:
Added check. Also implemented last chance to scroll to the parent element if we have some problem in the URL or can't find element